ES/ESdd017.xml

```diff
@@ -202,232 +228,194 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
-                        <item xml:id="e3"><desc>Names
-                           of the scribes are written in the bottom margin, on the first and last
-                           fols. of the respective text portion. </desc></item>
-                        <item xml:id="e4"><desc>Omitted ʾarke-hymns and
-                           fragments of commemorative notices are written in the upper margin, in
-                           one of the main hands: <locus target="#4rc">4rc</locus>, 4vbc, 5ra, 32rc,
-                           42vab, 60r, 83r, 97v, 105r, 105v, 122rc, 123vc, 124r.</desc></item>
-                        <item xml:id="e5"><desc>Omitted
-                           ʾarke-hymn is crudely written in the upper margin, in a secondary hand:
-                           42vc-43ra. </desc></item>
-                        <item xml:id="e6"><desc> Numerous corrections are written over erasures: e.g.,
-                              <locus target="#7rb">7rb</locus>, 12vb, 19rc, 34rb, 49rb, 107rb,
-                           112rc, 131rab, 149rabc, 165rab.</desc></item>
-                        <item xml:id="e7"><desc>Writing exercises: <locus target="#183vb">183vb</locus>, 184ra.</desc></item>
-                        <item xml:id="e8"><!--2019-04-02+02:00 Updated DD-017 
-            from DomLib -->
-                           <desc type="findingAid">- Names of the months are written in the centre of
-                              the upper margin: Maskaram: fols. 2r, 10r, 18r, 26r Ṭǝqǝmt: fols. 32r,
-                              34r, 42r, 50r Ḫǝdār: fols. 59r, 66v, 67r, 74v, 75r Tāḥśāś: fols. 87r,
-                              95r, 103r, 111r, 119v Ṭǝrr: fols. 120r, 128r, 136r, 144r, 152r
-                              Yakkātit: fols. 158r, 160r, 168r, 176r.</desc>
+                           <locus target="#184r"/>
+                           <desc type="CustomaryLaw"><foreign xml:lang="gez">Nagara wagʾ</foreign>, note concerning customary law (?). The note is partly illegible.</desc>
+                           <q xml:lang="gez">አኰቴተ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩፡ አምላክ፡ ናሁ፡ ጸሓፍነ፡ ነገረ፡ ወግእ፡ ዘገብረ፡ <supplied reason="lost">ቴ</supplied>ዎድሮስ፡ ወልደ፡ ፋኑኤል፡<gap reason="ellipsis" resp="MK"/> </q>
                         </item>
-                        <item xml:id="e9"><locus target="#2r">2r</locus><desc>Shelfmark <idno>C3-IV-225</idno>.</desc></item>
-                        <item xml:id="e10"><locus target="#1r">1r</locus><desc>Shelfmark <idno>364</idno>.</desc></item>
                         
+                        <item xml:id="e1">
+                           <desc>The title of the work, <foreign xml:lang="gez">ስንክሳር</foreign>, is written on the outer face of the front board.</desc></item>
+                        
+                        <item xml:id="e2">
+                           <locus target="#1r"/>
+                           <desc>The internal number of the monastic library, <idno>384</idno>.</desc></item>
```

Comment:
I have never seen `<idno>` encoded here; it is better to ask Dorothea or Eugenia whether it is possible. If it is, we can all adopt this practice :-)  otherwise we add all such identifiers to the top, where we encode `<msIdentifier>´, as `altIdentifier.`